test/suites/zombie-tanssi-relay/test-tanssi-relay.ts

```diff
@@ -1,21 +1,442 @@
+
+            const relayNetwork = relayApi.consts.system.version.specName.toString();
+            expect(relayNetwork, "Relay API incorrect").to.contain("starlight");
+
+            const container2000Network = container2000Api.consts.system.version.specName.toString();
+            const paraId2000 = (await container2000Api.query.parachainInfo.parachainId()).toString();
+            expect(container2000Network, "Container2000 API incorrect").to.contain("container-chain-template");
+            expect(paraId2000, "Container2000 API incorrect").to.be.equal("2000");
+
+            const container2001Network = container2001Api.consts.system.version.specName.toString();
+            const paraId2001 = (await container2001Api.query.parachainInfo.parachainId()).toString();
+            expect(container2001Network, "Container2001 API incorrect").to.contain("frontier-template");
+            expect(paraId2001, "Container2001 API incorrect").to.be.equal("2001");
+
+            const container2002Network = container2002Api.consts.system.version.specName.toString();
+            const paraId2002 = (await container2002Api.query.parachainInfo.parachainId()).toString();
+            expect(container2002Network, "Container2002 API incorrect").to.contain("container-chain-template");
+            expect(paraId2002, "Container2002 API incorrect").to.be.equal("2002");
+
+            // Test block numbers in relay are 0 yet
+            const header2000 = await getHeaderFromRelay(relayApi, 2000);
+            const header2001 = await getHeaderFromRelay(relayApi, 2001);
+            const header2002 = await getHeaderFromRelay(relayApi, 2002);
+
+            expect(header2000.number.toNumber()).to.be.equal(0);
+            expect(header2001.number.toNumber()).to.be.equal(0);
+            expect(header2002.number.toNumber()).to.be.equal(0);
+        }, 120000);
+
         it({
             id: "T01",
             title: "Blocks are being produced on tanssi-relay",
             test: async function () {
-                console.log("executing");
-                const relayApi = context.polkadotJs("Tanssi-relay");
                 const relayNetwork = relayApi.consts.system.version.specName.toString();
                 expect(relayNetwork, "Relay API incorrect").to.contain("starlight");
                 const blockNum = (await relayApi.rpc.chain.getBlock()).block.header.number.toNumber();
                 expect(blockNum).to.be.greaterThan(0);
             },
         });
+
+        it({
+            id: "T03",
+            timeout: 600000,
+            title: "Test assignation did not change",
+            test: async function () {
+                // TODO: starlight collator assignment is not set properly on genesis, so we need to wait 2 sessions
+                // for the collators to be assigned, and a bit more for container chains to start producing blocks
+                await waitSessions(context, relayApi, 4, null, "Tanssi-relay");
```

Comment:
TODO: Set this back down to 3, or even 2 if it works

Reply:
yes please!